algoliasearch-core/src/test/java/com/algolia/search/integration/TestHelpers.java

```diff
@@ -19,4 +28,34 @@ public static String getMcmUserId() {
     return String.format(
         "java-%s-%s", DateTimeFormatter.ofPattern("yyyy-MM-dd-HH-mm-ss").format(utc), userName);
   }
+
+  public static void checkEnvironmentVariable() throws Exception {
+    if (ALGOLIA_APPLICATION_ID_1 == null || ALGOLIA_APPLICATION_ID_1.isEmpty()) {
+      throw new Exception("ALGOLIA_APPLICATION_ID is not defined or empty");
```

Comment:
Following the previous comment about renaming all variables with the same name as their respective environment variable, I'd suggest to also change the name accordingly in all those thrown exceptions to prevent any confusion.